samples/shields/x_nucleo_53l0a1/prj.conf

```diff
@@ -0,0 +1 @@
+# nothing here
```

Comment:
You can drop this commit, fixed by https://github.com/zephyrproject-rtos/zephyr/pull/55973

Reply:
Rebased and drop it.
Thank you for notifiy.